base/string_utils.cpp

```diff
@@ -326,4 +326,29 @@ bool AlmostEqual(string const & str1, string const & str2, size_t mismatchedCoun
   return false;
 }
 
+void Split(string const & s, char delimiter, vector<string> & target)
```

Comment:
Не проходит на сплит пустой строки так как возвращает пустую строку. Но
если добавить if то будет проходить. Остальные два проходит.
On Jun 21, 2016 10:19 PM, "Ilya Zverev" <notifications@github.com> wrote:

> In base/string_utils.cpp
> <https://github.com/mapsme/omim/pull/3625#discussion_r67934285>:
>
> > @@ -326,4 +326,29 @@ bool AlmostEqual(string const & str1, string const & str2, size_t mismatchedCoun
> >    return false;
> >  }
> >
> > +void Split(string const & s, char delimiter, vector<string> & target)
>
> Спасибо, что проверил.
> Проходит ли твой вариант функции тест из string_utils_test?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/mapsme/omim/pull/3625/files/d3b23516fe8a4e51f454f05ffd552d299b02ca9b#r67934285>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAbvp2B_JJnQb0tOglB6dbalkS4L13fZks5qODlNgaJpZM4I647h>
> .
>